entries/w/webcentral.com.au.json

```
@@ -0,0 +1,13 @@
+{
+  "Webcentral": {
+    "domain": "webcentral.com.au",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://support.webcentral.com.au/s/article/Multi-Factor-Authentication-Guide",
+    "keywords": [
+      "domains",
+      "hosting"
+    ]
```

I'd still like a region tag here as they market themselves as **the** registrar/host for Australians.
```suggestion
    ],
    "regions": [
      "au"
    ]
```